integrations/access/servicenow/client.go

```diff
@@ -250,6 +260,24 @@ func (snc *Client) CheckHealth(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 	defer resp.RawResponse.Body.Close()
+
+	defer func() {
```

Comment:
Why do we need to defer the `Emit` operation?

Reply:
We dont, have removed the defer. Can't remember why i did that.